docs/userguide/envvars.md

```diff
@@ -0,0 +1,103 @@
+# Environment Variables
+
+Several aspects of BayBE can be configured via environment variables.
+
+## Basic Instructions
+Setting an environment variable with the name `ENVVAR_NAME` is best done before calling
+any Python code, and must also be done in the same session unless made persistent, e.g.
+via `.bashrc` or similar.
+```bash
+ENVAR_NAME="my_value"
+python do_baybe_work.py
+```
+Note that variables set in this manner are interpreted as text, but converted internally
```

Comment:
Is this note necessary? For me, this feels like a detail that is not relevant for a user.

Reply:
if you read on it mentions the important case of boolean which can be provided in several ways beyond "0" or "1", hence this general mention